frameworks/Haskell/servant/benchmark_config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "framework": "servant",
+  "tests": [{
+    "default": {
+      "setup_file": "setup",
+      "json_url": "/json",
+      "db_url": "/db",
+      "query_url": "/queries?queries=",
+      "fortune_url": "/fortune",
+      "update_url": "/updates?queries=",
+      "plaintext_url": "/plaintext",
+      "port": 7041,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "None",
+      "framework": "servant",
+      "language": "Haskell",
+      "orm": "Raw",
+      "platform": "Wai",
+      "webserver": "Warp",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "servant",
+      "notes": "",
+      "versus": ""
+    }
+  }]
```

Comment:
@jkarni would you mind updating this:
`"framework": "None"`
and also under framework add:
`"flavor": "GHC710"

I know it's a bit confusing, but framework is for whether or not your framework is using a framework. I have an open PR to start updating other benchmark configs and will be updating the docs soon.

All tests are good on local environment and will merge as soon as you can update these. Thanks!